CONTRIBUTING.md

```diff
@@ -1,71 +1,112 @@
+            needs to be dynamically added to the UI by a Dash callback in `demo_callbacks.py`, it
+            is encouraged to add a function generating this component to `demo_interface.py` and
+            call this function from `demo_callbacks.py`.
+        *   [demo_configs.py](demo_configs.py) contains all configurations and settings for the
+            demo and is intended as a quick way to customize the demo for a specific audience or
+            use case. It is encouraged to add more customization settings to this file; however,
+            this file should not be cluttered with settings that are not helpful to the end user
+            (eg titles and other settings that are unlikely to ever be changed).
+        *   [demo.css](demo.css) contains all custom CSS styling for the demo; any new CSS rules
+            should be added here or in a new file in the `/assets/` directory. Dash reads all
+            files in `/assets/` in alphabetical order so, out of the provided CSS files, `demo.css`
+            runs last and can overwrite previous styling rules.
+        *   [__demo_variables.css](__demo_variables.css) contains all CSS variables.
+            It is encouraged to edit or add more CSS variables here.
+        *   [demo_enums.py](demo_enums.py) contains [Enum](https://docs.python.org/3/library/enum.html)
+            classes for any settings or variables are being used frequently to avoid string
+            comparisons or other fragile code practices.
+
+    2.  All new files should be added to one of the following directories:
+        *   `/assets/` is a Dash-specific directory that must not contain subdirectories and must
+            not be renamed. Dash always reads files in this directory in alphabetical order and
+            therefore you do not need to explicitly import files from this directory. All CSS files
+            that **do not** contain the word `demo` in the title, should _ideally_ not be altered.
+        *   `/src/` should contain all functional code for the demo including solver implementations,
+            class definitions, etc. You can add subdirectories as needed.
+        *   `/static/` should contain all new static files such as images. You can add subdirectories
+            as needed.
+        *   `/tests/` should contain tests for all code found in `/src/`. All tests should be
+            discoverable through `python -m unittest discover`.
+
+    3.  Follow the [README template](README.md) to create a README that describes your example and
+        outlines how it works.
+
+3.  Once your example has been implemented,
+    [open an issue](https://github.com/dwave-examples/template-dash/issues/new/choose) in the
+    [dwave-examples/template-dash](https://github.com/dwave-examples/template-dash) repository
+    with a link to your example.
+
+4.  D-Wave will fork your repository and may make a pull request to get your approval for certain
+    code changes.
```

Comment:
```suggestion
When adding a contributed example, D-Wave forks your repository and may make a pull 
request to get your approval for certain code changes.
```

"D-Wave will fork your repository" comes across a bit like a blanket guarantee to accept the contribution. 

Also, this is not a step they contributor needs to take as part of the process so perhaps not a numbered paragraph?